types/react-native/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//                 Naoufal El Yousfi <https://github.com/nelyousfi>
```

Comment:
Just want to make sure that you understand that adding your name here means you’re signing up as maintainer and will get mentions on all future issues and PRs for the react-native typings. Is that what you intended?